modules/installation-cloudformation-dns.adoc

```diff
@@ -9,6 +9,16 @@
 You can use the following CloudFormation template to deploy the networking
 objects and load balancers that you need for your {product-title} cluster.
 
+[IMPORTANT]
+====
+If you are deploying the networking objects and load balancers, you must ensure the IngressController's public and private subnets are correctly tagged for Kubernetes to pick the correct internal load balancer. For example:
+[source,yaml]
+----
+RegisterPublicSubnetTags: http://kubernetes.io/role/elb
+RegisterPrivateSubnetTags: kubernetes.io/role/internal-elb
```

Comment:
I prefer something like:

Register public subnet tags:
```
kubernetes.io/cluster/<infra-id>: shared or owned
```
Register private subnet tags:
```
kubernetes.io/cluster/<infra-id>: shared or owned
kubernetes.io/role/internal-elb: <empty>
```

@lihongan WDYT? refer to https://issues.redhat.com/browse/OCPQE-10362?focusedCommentId=20332409&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-20332409